### ISSUE TITLE
Maven build log polluted with Assembly plugin redundant logs

### ISSUE THREAD

**Issue by stormdb** [yermak/AudioBookConverter#108]
Maven build log is heavily polluted with redundant Assembly plugin messages:
```
[INFO] META-INF/ already added, skipping
[INFO] META-INF/MANIFEST.MF already added, skipping
[INFO] org/ already added, skipping
```
Old version of Assembly plugin used by default has a bug related to logging.

**Comment by stormdb** [comment]
Fixed in PR #109

**Comment by stormdb** [comment]
Fixed in v5.4.0
